examples/plot_bands_functionality.py

```diff
@@ -0,0 +1,107 @@
+"""
+Plot Bands of Satellite Imagery with EarthPy
+==================================================================
+
+Learn how to plot each individual band from satellite imagery with
```

Comment:
@nkorinek this function is not just for satellite imagery. we want vignettes t be generic. this means we just want to show people how to use the function. the data could be from lots of raster sourcess.

Reply:
Learn how to use the EarthPy `plot_bands()` function to quickly plot a single or many raster bands of an image.  Plot_bands() can also be used to plot single raster layers with legends.